src/test_hash_table.py

```diff
@@ -0,0 +1,35 @@
+"""Tests for Hash Table."""
+from hash_table import HashTable
+import pytest
+import io
+
+
+@pytest.fixture()
+def word_file():
+    file = io.open('/usr/share/dict/words')
+    words = file.read()
+    file.close
+    dic = {}
+    for word in words:
+        dic[word] = len(dic)
+
+
```

Comment:
You need to test the size of the hash table, test that an improper key throws an error, test for collisions, as well as other things. Take another swing at this.